src/useQueryParam.ts

```diff
@@ -53,10 +62,10 @@ export const useQueryParam = <D, D2 = D>(
     // value may be an array that is recreated if a different query param
     // changes.
   }, [
-    encodedValue instanceof Array
-      ? stringify({ name: encodedValue })
-      : encodedValue,
-  ]);
+      encodedValue instanceof Array
+        ? stringify({ name: encodedValue })
+        : encodedValue,
+    ]);
```

Comment:
I guess I need to add prettier to the repo configuration. oops

Reply:
Would probably pair that with TSLint.